test/common/network/utility_fuzz_test.cc

```diff
@@ -0,0 +1,85 @@
+#include "envoy/config/core/v3/address.pb.h"
+
+#include "common/network/address_impl.h"
+#include "common/network/utility.h"
+
+#include "test/fuzz/fuzz_runner.h"
+
+namespace Envoy {
+namespace Fuzz {
+
+DEFINE_FUZZER(const uint8_t* buf, size_t len) {
+  const std::string string_buffer(reinterpret_cast<const char*>(buf), len);
+
+  {
```

Comment:
These braces for scope are redundant. It looks like, throughout, you could just move all the local variables into the `try` blocks and get rid of one level of nesting.

Reply:
Yeah, that definitely makes the code cleaner. I removed the unnecessary braces and moved everything into the `try` blocks.